src/base/bittorrent/loadtorrentparams.h

```diff
@@ -28,6 +28,8 @@
 
 #pragma once
 
+#include <optional>
```

Comment:
why add it?

Reply:
> why add it?

The remainder of some previous iteration... Deleted it.